lib/i18n/backend/chain.rb

```diff
@@ -92,9 +92,13 @@ def init_translations
           end
 
           def translations
-            backends.first.instance_eval do
-              init_translations unless initialized?
-              translations
+            backends.reverse.inject({}) do |h, b|
```

Comment:
Please use `each_with_object` here (as it is mentally parse / to read than `inject`), as well as more descriptive variable names than `h` and `b`.